src/components/atoms/Identicon/Identicon.tsx

```diff
@@ -0,0 +1,19 @@
+import * as jdenticon from 'jdenticon';
+import React, { useEffect, useRef } from 'react';
+
+interface IdenticonProps {
+    value: string;
+    size: number;
+}
+
+export const Identicon: React.FC<IdenticonProps> = ({ value, size }) => {
```

Comment:
@biga816 we avoid using React.FC.
export const Identicon = ({value, size} : IdenticonProps) => {}

Reply:
I fixed it!